resources/metadata/nexusUpload.yaml

```diff
@@ -116,12 +116,20 @@ spec:
         scope:
         - PARAMETERS
         secret: true
+        resourceRef:
+          - name: nexusCredentialsId
+            type: secret
+            param: username
```

Comment:
The param is named "user", not "username". (Which is an inconsistency on my account which I should fix.)

Reply:
see #1960

Reply:
@lndrschlz this refers to the `username` parameter of Jenkins credentials, not the step parameter right?

Reply:
[This](https://github.com/SAP/jenkins-library/blob/CCFenner/credentialLink/vars/nexusUpload.groovy#L16) is where it needs to be aligned. Which may or may not have anything to do with the above. :-}

Reply:
Ah, good catch! Haven't thought about that.